<launch>
    <node name="robotis_nodelet_manager" pkg="nodelet" type="nodelet" args="manager" />
    <node name="robotis_manager_nodelet" pkg="nodelet" type="nodelet" args="load robotis_controller/RobotisManagerNodelet robotis_nodelet_manager" output="screen">
        <rosparam command="load" file="$(find robotis_example)/launch/ROBOTIS-OP2.yaml" />
        <rosparam param="publish_joint_topic_name">"/robot_joint_states"</rosparam>
        <rosparam param="subscribe_joint_topic_name">"/controller_joint_states"</rosparam>
    </node>
</launch>